app/src/controllers/uniqueErrors/sagas.js

```diff
@@ -93,7 +94,7 @@ function* fetchClusters(payload = {}) {
     const uniqueErrorsParams = yield select(pagePropertiesSelector, NAMESPACE);
     requestParams.data = {
       launchId,
-      mode: uniqueErrorsParams.mode || ALL,
+      mode: uniqueErrorsParams.mode,
       pageNumber: query[PAGE_KEY],
```

Comment:
Why not `...uniqueErrorsParams` according to the previous comment?

Reply:
`uniqueErrorsParams` can contain many parameters not only mode, should we pass all of them?

Reply:
Why not if they are part of unique errors? It is better than expose some properties that hidden in plugin.